### PR TITLE
[Issue #1546] Deploy Analytics folder to AWS ECR

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary
Fixes https://github.com/HHS/simpler-grants-gov/issues/1546

## Changes proposed

1. Adds an AWS ECR repository into `infra/analytics`
2. Adds a `cd-analytics` script that I used to confirm that the ECR repository push was working

## Context for reviewers

The PR was like 95% copy pasting

## Testing

image

https://us-east-1.console.aws.amazon.com/ecr/repositories/private/315341936575/simpler-grants-gov-analytics?region=us-east-1

### CODE REVIEW

File: infra/analytics/build-repository/main.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
+data "aws_iam_role" "github_actions" {
```
Comment (coilysiren) [inline infra/analytics/build-repository/main.tf:1]: Copied from `infra/api/build-repository/main.tf`

File: .github/workflows/cd-analytics.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Deploy Analytics
```
Comment (coilysiren) [inline .github/workflows/cd-analytics.yml:1]: Copied from `.github/workflows/cd-api.yml`

File: .github/workflows/cd-analytics.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        options:
+          - dev
+          - staging
+          - prod
+
+jobs:
+  analytics-checks:
+    name: Run Analyics Checks
+    uses: ./.github/workflows/ci-analytics.yml
+    secrets: inherit
```
Comment (coilysiren) [inline .github/workflows/cd-analytics.yml:30]: Tests were broken before I added this line

```
    secrets: inherit
```

File: infra/analytics/app-config/outputs.tf [hunk 4]
```diff
@@ -17,3 +17,11 @@ output "build_repository_config" {
...
   value = local.environment_configs
 }
+
+# This variable is slightly misnamed. It should really be called "has_migrations".
+# It controls whether or not the `run-database-migrations.sh` script tries to run database
+# migrations. The entire analytics application is going to have its schema controlled
+# via ETL jobs, so we don't need to run migrations in the same way as the API.
+output "has_database" {
+  value = false
+}
```
Comment (coilysiren) [inline infra/analytics/app-config/outputs.tf:27]: I don't intend on updating the variable name. It seems fine to just document the issue here.

**Review by aplybeah — APPROVED** [review]
LGTM

**Pull request merged**